doctr/models/predictor/tensorflow.py

```diff
@@ -33,6 +34,8 @@ class OCRPredictor(NestedObject, _OCRPredictor):
             accordingly. Doing so will improve performances for documents with page-uniform rotations.
         detect_orientation: if True, the estimated general page orientation will be added to the predictions for each
             page. Doing so will slightly deteriorate the overall latency.
+        detect_language: if True, the language prediction will be added to the predictions for each
```

Comment:
It can only detect the same language per page. So for a document with multiple pages it's possible to have multiple languages. you think we should add the "s" ? and should we add it also for orientation ?

Reply:
ok than without "s" :) I have had in mind that we get probs for possible languages but if it is only one than without "s"